ruby-challenges.rb

```diff
@@ -2,15 +2,25 @@
 # MINASWAN ✌️
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
-
+# I kept getting stuck becuase I put 'p #{num}  is even"' and 'p #{num}  is odd' when that line isn't supposed to have a print.
 num1 = 7
 # Expected output: '7 is odd'
 num2 = 42
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
 
+def even_odd num
+    if num % 2 == 0
```

Comment:
To dig into the expansive built in methods available in Ruby, check out the .even? or .odd? method.